src/DependencyInjection/Compiler/ExtensionCompilerPass.php

```diff
@@ -129,41 +129,24 @@ private function getExtensionsForAdmin(string $id, Definition $admin, ContainerB
     /**
      * Resolves the class argument of the admin to an actual class (in case of %parameter%).
      *
-     * @phpstan-return class-string|null
+     * @phpstan-return class-string
      */
-    private function getManagedClass(Definition $admin, ContainerBuilder $container): ?string
+    private function getManagedClass(Definition $admin, ContainerBuilder $container): string
```

Comment:
signature and phpdoc changed because this method no longer returns null (it throws exceptions instead)